packages/v4/patternfly-docs/components-data.json

```diff
@@ -360,6 +360,6 @@
   },
   "wizard": {
     "illustration": "./images/component-illustrations/wizard.png",
-    "summary": "A <b>wizard</b> provides a guided workflow that offers a path to complete a task, create an object or objects, or finish a series of steps for some other outcome. Wizards should incite trust in the user and guide them through an otherwise overwhelming experience.<br/><br/>A newer React implementation of the wizard will replace the existing implementation at the next major release. The documentation for the newer implementation is under the <Link to='/components/wizard/react-next'>React next</Link> tab, and this newer implementation can be imported from @patternfly/react-core/next."
+    "summary": "A <b>wizard</b> provides a guided workflow that offers a path to complete a task, create an object or objects, or finish a series of steps for some other outcome. Wizards should incite trust in the user and guide them through an otherwise overwhelming experience.<br/><br/>A newer React implementation of the wizard will replace the existing implementation at the next major release. The documentation for the newer implementation is under the <Link to='/components/wizard/react-deprecated'>React next</Link> tab, and this newer implementation can be imported from @patternfly/react-core/next."
```

Comment:
This summary is now outdated as well. Could you update it to read:
```
"A <b>wizard</b> provides a guided workflow that offers a path to complete a task, create an object or objects, or finish a series of steps for some other outcome. Wizards should incite trust in the user and guide them through an otherwise overwhelming experience.<br/><br/>A newer React implementation of the wizard has replaced the now deprecated implementation. The documentation for the deprecated implementation is under the <Link to='/components/wizard/react-deprecated'>React deprecated</Link> tab, and this deprecated implementation can be imported from @patternfly/react-core/deprecated."
```